react/theme.css

```diff
@@ -10,3 +10,8 @@ html, body, #render-container, #render-provider, #render-provider > article {
 body {
   background-color: white;
 }
+
+input[type=number]::-webkit-inner-spin-button,
+input[type=number]::-webkit-outer-spin-button {
+   opacity: 1;
```

Comment:
The class `o-100` from `vtex-tachyons` has a similar behaviour.
https://github.com/vtex/vtex-tachyons/blob/master/tachyons.css#L1304

Reply:
Done!